### DISCUSSION TITLE
Is it possible to switch from the side nav bar to a top nav bar?

### DISCUSSION THREAD

**Discussion by wcoy7** [filamentphp/filament#10224] [Help] [answered]
I tried to search for nav bar or "top nav bar" and the results really didn't match.  My apologies if this topic is somewhere, I couldn't find it.

**Comment by devhoussam1998** [comment, marked as answer]
of course you just need to add `->topNavigation()` inside your panel provider

check this out: https://filamentphp.com/docs/3.x/panels/navigation#using-top-navigation

**Reply by newtec-foggia** [reply]
Is it possible based on the user with the same panel?

**Comment by wcoy7** [comment]
Mind blown, ever time I think something is not doable it turns out it's either baked in or a plugin is available.  Amazing platform.
